### PR TITLE
Created EnvConfigTests

### PR DESCRIPTION
**Pull request by patrykcalka**
Created 5 unit tests for EnvConfig class. These tests cover base functionality of EnvConfig class. Issue #51

### CODE REVIEW

**Review by agatamrnw — DISMISSED** [review]

File: src/test/java/com/github/splendor_mobile_game/websocket/config/EnvConfigTests.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        // 1. Setup data for the test
+        int port = 8887;
+        int connectionLostTimeoutSec = 120;
+        int pingIntervalMs = 1000;
+        int connectionCheckIntervalMs = 2000;
+        String logsDir = "./logs/";
+
+        // 2. Call the function you are testing.
+        try {
+            EnvConfig config = new EnvConfig();
```
Comment (kestivvi) [inline src/test/java/com/github/splendor_mobile_game/websocket/config/EnvConfigTests.java]: Unfortunately you should not check for the default config which is in the root directory, simply because it will change a lot.

**Review by kestivvi — CHANGES_REQUESTED** [review]
Great simple tests! 🎉🎉 

I would add also test checking what if some value is empty ie.
```env
PORT=
CONNECTION_LOST_TIMEOUT_SEC=100
PING_INTERVAL_MS=1000
CONNECTION_CHECK_INTERVAL_MS=1000
LOGS_DIR=./logs/
```

and what if in the config file there are some additional fields ie.
```env
PORT=8887
a=312
CONNECTION_LOST_TIMEOUT_SEC=100
bc=dsa
PING_INTERVAL_MS=1000
CONNECTION_CHECK_INTERVAL_MS=1000
LOGS_DIR=./logs/
```

I plan in the distant future to refactor EnvConfig class, so it does not receive path to the file but simply just `FileHandler` or something so creating files wouldn't be necessary.

**Review by kestivvi — APPROVED** [review]
What an awesome test!! Great work! 🎊🎉✨
